doc/src/manual/environment-variables.md

```diff
@@ -156,6 +156,15 @@ The absolute path `REPL.find_hist_file()` of the REPL's history file. If
 $(DEPOT_PATH[1])/logs/repl_history.jl
 ```
 
+### `JULIA_PKG_SERVER`
+
+Used by `Pkg.jl`, for loading packages. By default, `Pkg` uses https://pkg.julialang.org to
+host julia packages. You can use this environment variable to disable the use of this
+package server, and instead access the packages hosted by GitHub, by setting:
```

Comment:
I'm very far from being an authority in the Pkg land, but I think "loading" and "hosting" packages are used incorrectly here.  I'd say something along these lines
```suggestion
Used by `Pkg.jl`, for downloading packages and updating the registry. By default, `Pkg` uses `https://pkg.julialang.org` to
fetch Julia packages. You can use this environment variable to select a different server, or also to disable the use of the
PkgServer protocol, and instead access the packages directly from their hosts (GitHub, GitLab, etc.) by setting:
```

Reply:
Cool, thanks for the change!